examples/Utils/ArgumentNames.php

```diff
@@ -81,6 +86,9 @@ final class ArgumentNames
         self::CRITERION_ID => 'The criterion ID',
         self::CUSTOMER_ID => 'The customer ID without dashes',
         self::DRAFT_ID => 'The draft ID',
+        self::FEED_ID => 'The feed ID',
+        self::FEED_ITEM_ID => 'The feed item ID',
+        self::FLIGHT_PLACEHOLDER_FIELD_NAME => 'The flight placeholder field name',
```

Comment:
WDYT of just naming this FEED_PLACEHOLDER_FIELD_NAME? Then you can reuse it in other feed examples later.